stdlib/public/core/ManagedBuffer.swift

```diff
@@ -106,8 +106,10 @@ extension ManagedBuffer {
   /// Call `body` with an `UnsafeMutablePointer` to the stored
   /// `Header`.
   ///
-  /// - Note: This pointer is valid only for the duration of the
-  ///   call to `body`.
+  /// - Note: This pointer is fixed when the buffer is created, but it is only valid
```

Comment:
What does "fixed" mean here?  I'm guessing you mean "fixed" in the sense of being fixed in place, not in the sense of fixed vs broken.

The new docs are longer, but I'm not sure what facts were missing from the old version from following this PR's discussion and reading these changes.  The new version seems to suggest that it's valid to hold on to the buffer pointer after the end of the closure, if you do some work to explicitly extend its lifetime.  Is that legal?

Reply:
It remains unsafe to use these pointers outside the scoped accessor functions. The comment changes are meant to clarify that the pointers are expected to be stable over time (multiple calls to `with...()` on the same object should yield the same addresses.)

Reply:
What can/should a developer do, knowing that the pointer will be the same over time?  Or is this more of an "even though it might look safe, it isn't"?  In that case, how about:

> Although the pointer to the buffer doesn't change after the buffer is created,
> the pointer is valid only for the duration to the call to `body`.
> If you try to access the pointer outside of `body`,
> the buffer might have already been deallocated, causing a crash.

Reply:
One thing the stability of the pointer lets you do is pass it to C functions that expect it to remain stable, even if those function calls are made in separate `withUnsafePointerToBody` calls. The specific motivating example here is `os_unfair_lock`.

Reply:
Thanks — that's the missing fact!